<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="LoginGroupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>500</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 210, 184);
font: 10pt &quot;URW Gothic L&quot;;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QGroupBox" name="LoginCredentials">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>2) Enter your user ID :</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="userIDLabel">
        <property name="text">
         <string>User ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="userIDLineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="LoginType">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>1) Login as :</string>
    </property>
    <widget class="QRadioButton" name="staffRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>106</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Staff</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="clientRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>106</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Pet Adopter</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="cuacs_logo">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
